inc/classes/preload/class-full-process.php

```diff
@@ -85,14 +85,20 @@ protected function task( $item ) {
 	 * @return bool
 	 */
 	protected function is_already_cached( $item ) {
+		static $https;
+
+		if ( ! isset( $https ) ) {
+			$https = ( is_ssl() && get_rocket_option( 'cache_ssl' ) ) ? '-https' : '';
```

Comment:
`$https` is not used.
Maybe it should be `'index' . $https . '.html'` later then?